js/store.js

```diff
@@ -1,12 +1,12 @@
 // modules
-import arvic from './modules/arvic.js';
+import mikailashanin from './modules/mikailashanin.js';
```

Comment:
Write the file name of your module properly.

```suggestion
import mikailashanin from './modules/nesjohnjoelle.js';
```